### ISSUE TITLE
Hedera subspecies problem

### ISSUE THREAD

**Issue by timadriaens** [trias-project/indicators#92]
Hi, upon wanting to check emergence status of Atlantic ivy (Hedera hibernica), I noticed differences in taxon matching of several datases:

- Florabank matches with Hedera helix subsp. hibernica (Poit.) D.C.McClint. (gbif key 6307044) - considered a synonym by gbif
- observations.be matches with Hedera hibernica (G.Kirchn.) Carrière (gbif key 8168344), whereas in obs.be itself the taxon name is Hedera helix subsp. hibernica (Kirchner) McClintock

In GRISS Belgium, there is only Hedera hibernica (G.Kirchn.) Bean (such as in the Manual of Alien Plants) (gbif key 8410115). 

Fact: 3 different gbif keys for the same thing. Consequence: the species does not pop up as emerging in the trias indicator flow, whilst probably every field person will say it is clearly emerging.

Solution?

**Comment by damianooldoni** [comment]
Thanks @timadriaens to pointing out this.

[Hedera helix subsp. hibernica (Poit.) D.C.McClint.](https://www.gbif.org/species/6307044) is a synonym of [Hedera hibernica Poit.](https://www.gbif.org/species/8168344) so this two are by GBIF linked to each other :+1: 

The only problem is that [Hedera hibernica (G.Kirchn.) Bean](https://www.gbif.org/species/8410115) is not linked to the other two. @qgroom: do you think are there reasons to mention this to GBIF Backbone as an issue as we did in the past (see https://github.com/gbif/portal-feedback/issues/2025)? Obviously this doesn't mean we get an answer fast

**Comment by qgroom** [comment]
We can raise it as an issue. I suspect it will come down to the Catalogue of Life again and  getting that changed doesn't seem possible. Nevertheless, never give up never surrender...

**Comment by damianooldoni** [comment]
https://github.com/gbif/portal-feedback/issues/3225

**Comment by damianooldoni** [comment]
It seems that this issue is not an indicators issue in the sense that no one of these three taxa are present in the unified checklist.

While checking the last version of the occurrence cube metadata ([be_alientaxa_info.csv](https://zenodo.org/record/4299976/files/be_alientaxa_info.csv?download=1)) I fond that the only taxon from genus Hedera is _[Hedera colchica (K.Koch) K.Koch](https://www.gbif.org/species/3036033)_. It means these taxa are not present in any checklist we harvest or they are not taken into account during the process of unifying the cheklists. I will check this right now and I will open in case a new issue in other repo.

**Comment by damianooldoni** [comment]
I was partly wrong:
_[Hedera hibernica (G.Kirchn.) Bean](https://www.gbif.org/species/152544472)_ is present in the unified checklist.
But there are only 13 occurrences in BE and they are all preserved specimen from botanic garden Meise:
https://www.gbif.org/occurrence/search?country=BE&taxon_key=8410115 without coordinates (!) and 11 have no date even. So, no way they get to indicators. @qgroom: something to check within your colleagues of botanic garden, maybe?

Still, the other two taxa are not in the unified.

**Comment by timadriaens** [comment]
@damianooldoni yes, but the species is much more in the occurrences bit under different names, hence the problem for the indicators...

**Comment by timadriaens** [comment]
we merged the different dataset to try to build the GAM, see [this issue](https://github.com/inbo/fis-projecten/pull/54) , and if you do that you get of course a different picture of "emergence"

![image](https://user-images.githubusercontent.com/16116743/108190630-7cad1500-7112-11eb-980e-aa2d7f138919.png)

**Comment by damianooldoni** [comment]
Yes, @timadriaens, I agree on this. At the end it's an indicator problem. 

But the solution is to get one or both of the taxa 6307044 and 8168344 in the unified checklist. As I wrote this morning the taxa 8410115, present in GRIIS Belgium, doesn't play any role occurrence-speaking unless GBIF solves https://github.com/gbif/portal-feedback/issues/3225 which I opened 20 days ago and which I mentioned above, still unlikely it will happen shortly.

6307044 is synonym of 8168344. Still, it would be sufficient to have it published in one of the checklists and indicate we trust the synonymy match and the accepted taxon 8168344 will appear in the unified and will stream up to a new occurrence cube with all occurrences of both taxa.

**Comment by damianooldoni** [comment]
I have just asked an update status to GBIF: https://github.com/gbif/portal-feedback/issues/3225#issuecomment-780492693

**Comment by damianooldoni** [comment]
@timadriaens, update from GBIF Backbone:
- [Hedera hibernica Poit.](https://www.gbif.org/species/8168344) is the accepted taxon
- [Hedera hibernica (G.Kirchn.) Bean](https://www.gbif.org/species/8410115) is now pointing to Hedera hibernica Poit. (that's fine: searching for  Hedera hibernica Poit. would result in occs for both)
- [Hedera hibernica (G.Kirchn.) Carrière](https://www.gbif.org/species/3036157) is still considered apart (taxonomic status doubtful)
